docs/00_setup_guide.md

```diff
@@ -34,13 +33,16 @@ This new API stores all its contents inside a Postgres database.
 Version 9.4 or later is recommended because of the heavy reliance on JSON data
 types and operators.
 
-#### Neo4j
-A new, more powerful Neo4j-based graph search is enabled for certain entities and paths.
+#### Redis
 
-Development and test environments instances can be automatically installed and configured by the
-setup script.
+Redis is used to exchange messages between CER and the Gremlin indexer application.
+Version 4.0 or higher is recommended.
 
-Version 3.3 or higher is recommended.
+#### Gremlin Server
```

Comment:
there is a recommended version?

Reply:
Yes. We're using a fairly specific version, I'll update the docs.